scripts/codegen/ErrorId/generate_error_ids.py

```diff
@@ -107,15 +94,8 @@
             enum_members['non_critical'][board] = \
                 ['    %s_NON_CRITICAL_%s, \\' %(board, signal.name.upper()) for signal in can_msg.signals]
         except KeyError:
-            raise KeyError('Could not find non critical error message for %s' % board)
-
-        try:
-            can_msg = database.get_message_by_name(board + '_FAULTS')
-            enum_members['faults'][board] = \
-                ['    %s_FAULTS_%s, \\' %(board, signal.name.upper()) for signal in can_msg.signals]
-        except KeyError:
-            if board == "BMS":
-                raise KeyError('Could not find AIR shutdown error message for %s' % board)
+            if board != "BMS":
```

Comment:
again, no way to completely remove the BMS from the list of boards in the error table?

Reply:
The issue here is that BMS_AIR_SHUTDOWN_ERRORS is renamed to BMS_FAULTS where as the rest of the boards use AIR_SHUTDOWN_ERRORS, this band-aid won't been needed once the rest of the board's messages are re-named to _FAULTS. This can be done quickly after the WARNINGS rename...

Reply:
Yeah wouldn't recommend removing from `supported_boards` since that's currently used to error check cube and can code generation
As temporary solution this seems fine